### PR TITLE
Ladybugs

### PR DESCRIPTION
**Pull request by sonphantrung**
#### Summary
Content "Added lady bugs and replaced giant ladybugs with them"

#### Purpose of change
It's day one of Cataclysm, how can I see a giant ladybug?

#### Describe the solution
Added a mutated ladybug: Less resilient, less damaging. Replaced giant ladybugs (or toned down their spawn rate), with them

#### Describe alternatives you've considered
None

#### Testing
If it doesn't cause errors, it should be good

#### Additional context
Contributes to #55409

### CODE REVIEW

File: data/json/monstergroups/wilderness.json [hunk 1]
```diff
@@ -1814,7 +1814,7 @@
       { "monster": "mon_nakedmolerat_giant", "weight": 1, "cost_multiplier": 3, "conditions": [ "DUSK", "NIGHT" ] },
       { "monster": "mon_mutant_experimental", "weight": 3, "cost_multiplier": 0, "conditions": [ "DUSK", "NIGHT" ] },
       { "monster": "mon_mantis_small", "weight": 8, "cost_multiplier": 10 },
-      { "monster": "mon_lady_bug_giant", "weight": 5, "cost_multiplier": 10 },
```
Comment (Maleclypse) [inline data/json/monstergroups/wilderness.json:1817]: Add this spawn back with the start time

File: data/json/monstergroups/wilderness.json [hunk 2]
```diff
@@ -2887,7 +2887,7 @@
       { "monster": "mon_squirrel", "weight": 50, "cost_multiplier": 0 },
       { "monster": "mon_squirrel_red", "weight": 50, "cost_multiplier": 0 },
       { "monster": "mon_mantis_small", "weight": 5, "cost_multiplier": 10 },
-      { "monster": "mon_lady_bug_giant", "weight": 8, "cost_multiplier": 10 },
```
Comment (Maleclypse) [inline data/json/monstergroups/wilderness.json:2890]: Same as the last removed spawn

**Review by Maleclypse — CHANGES_REQUESTED** [review]

File: data/json/monstergroups/wilderness.json (hunk 1 above)
Comment (sonphantrung) [reply data/json/monstergroups/wilderness.json:1817]: I thought ladybugs evolve into giant ones with the halflife of 14 days?

**Review by Maleclypse — APPROVED** [review]
Thanks!

File: data/json/items/comestibles/egg.json [hunk 3]
```diff
@@ -352,6 +352,18 @@
...
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_lady_bug_giant",
+    "name": { "str": "giant lady bug egg" },
+    "copy-from": "egg_insect",
+    "color": "red",
+    "symbol": "o",
+    "spoils_in": "7 days",
+    "description": "A giant lady bug egg.  It looks like a huge yellow rice grain.  Inside there is a hungry giant translucent lavrae.",
```
Comment (LyleSY) [inline data/json/items/comestibles/egg.json:363]:
```suggestion
    "description": "A giant ladybug egg.  It looks like a huge yellow rice grain.  Inside there is a hungry giant translucent lavrae.",
```

File: data/json/items/comestibles/egg.json [hunk 4]
```diff
@@ -352,6 +352,18 @@
     "rot_spawn": "GROUP_EGG_LADY_BUG",
     "rot_spawn_chance": 50
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_lady_bug_giant",
+    "name": { "str": "giant lady bug egg" },
```
Comment (LyleSY) [inline data/json/items/comestibles/egg.json:358]:
```suggestion
    "name": { "str": "giant ladybug egg" },
```

This is standard spelling in the US https://en.wiktionary.org/wiki/ladybug

**Review by LyleSY — CHANGES_REQUESTED** [review]
This is mostly really good. Not sure what you intend with aggression right now. Now at 20 spaces it will see you and immediately want to murder you. When you get close it will want to murder you more (not sure how much), then once you hurt it it will want to murder you but slightly less (not sure how much), then run away when it has 48 or less HP.  Is that the intent?

File: data/json/monsters/insect_spider.json [hunk 5]
```diff
@@ -2635,6 +2635,96 @@
...
+  {
+    "id": "mon_lady_bug",
+    "type": "MONSTER",
+    "name": { "str": "lady bug" },
+    "description": "This is a mutated lady bug the size of a dog.  While not as dangerous as her giant sisters, she's still troublesome to deal with.",
+    "default_faction": "ladybug",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "200 L",
+    "weight": "300 kg",
```
Comment (NetSysFire) [inline data/json/monsters/insect_spider.json:2647]: >the size of a dog

>`"weight": "300 kg",
"volume": "200 L",`

That seems off.

**Comment by sonphantrung** [comment]
Now, the ladybug's aggression is like the giant rattlesnake

**Review by LyleSY — APPROVED** [review]
Big improvement, fits much better with the game, makes sense

**Pull request merged**